src/Components/Aspire.Azure.AI.Inference/ChatCompletionsClientSettings.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Core;
+using System.Data.Common;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Represents configuration settings for Azure AI Chat Completions client.
+/// </summary>
+public sealed class ChatCompletionsClientSettings
+{
+    /// <summary>
+    /// Gets or sets the name of the AI model to use for chat completions.
+    /// </summary>
+    public string? ModelName { get; set; }
```

Comment:
Where is this used?

Reply:
Possibly isn't, I think it was from an older implementation